runtime/dancebox/src/lib.rs

```diff
@@ -940,6 +944,16 @@ impl InstanceFilter<RuntimeCall> for ProxyType {
             ProxyType::Balances => {
                 matches!(c, RuntimeCall::Balances(..) | RuntimeCall::Utility(..))
             }
+            ProxyType::Registrar => {
+                matches!(c, RuntimeCall::Registrar(..) | RuntimeCall::Utility(..))
+            }
+            ProxyType::SudoRegistrar => {
+                matches!(
+                    c,
+                    RuntimeCall::Sudo(pallet_sudo::Call::sudo {call: ref x})
+                    if matches!(x.as_ref(), RuntimeCall::Registrar(..))
```

Comment:
Compared to SudoBalances this is missing RuntimeCall::Utility, but I guess that's good, we don't want the proxy to be able to call sudo(utility.dispatchAs)... But what about allowing utility.batch*, that should be ok?

Reply:
I am actually not entirely sure what would happen if we enable utility here, but I see that parity has it enabled in a similar proxy type: https://github.com/paritytech/polkadot-sdk/blob/4c4407a8930267e139db93019c01444acf49e1ce/polkadot/runtime/westend/src/lib.rs#L1066

@nanocryk can you try it out?

Reply:
I was wrong, it depends on how you define the proxy:

`Sudo(Registrar) || Utility` this is fine
`Sudo(Registrar || Utility)` this is bad

The current way of always allowing utility should be ok as well